quicktests/overlaying/tests/functional/formatter.js

```diff
@@ -12,7 +12,17 @@ function run(svg, data, Plottable) {
     };
 
 
-  var big_numbers = data[0].slice(0, 5);
```

Comment:
This file was doing a shallow copy, and then manipulating data. Because the data was manipulated twice, the two quicktests looked different from each other, which made them difficult to compare. I deleted .slice() from a lot of files in favor of deep copies.